### PR TITLE
fix for codeql?

### PR DESCRIPTION
**Pull request by Zemurin**

### CODE REVIEW

**Review by IhateTrains — APPROVED** [review]
Frankly I don't care about Linux build as long as it's green, but the code seems correct.

File: LinuxUtils.cpp [hunk 1]
```diff
@@ -370,8 +370,9 @@ class ConversionOutputBuffer
 		static void str(String& output, char* buffer, std::size_t length)
 		{
 			using namespace std;
```
Comment (IhateTrains) [inline LinuxUtils.cpp:372]: I think it's not needed anymore.

Comment (Zemurin) [reply LinuxUtils.cpp:372]: It's all over the place. Since this file is ... `special`, I'm not touching it with a 20-foot pole.

**Review by Idhrendur — APPROVED** [review]
I generally don't have much of an idea what the linux utils are trying to do or how they're going about it. This looks to be correct.

If not, we'll need a linux dev to come by and make things better.

**Pull request merged**